eventdiscovery-sdk/src/main/java/com/schedjoules/eventdiscovery/framework/microfragments/eventdetails/fragments/views/EventHeaderDateTimeView.java

```diff
@@ -46,6 +46,8 @@ public EventHeaderDateTimeView(SchedjoulesDetailsHeaderBinding binding)
     @Override
     public void update(Event event)
     {
+        // TODO: go back to this version once the API stops to return "estimated" end times:
+        // CharSequence rangeCharSequence = new DateTimeRange(event).value(mDateTimeViewCollapsed.getContext());
```

Comment:
This is not really about the `Duration` being present or not. `DateTimeRange` actually handles absent and present `Duration`s.
At present the API always returns a default duration of 2 hours, but that's wrong in most cases. So we currently use `LongDateAndTimeNoYear` which doesn't show an end time at all (even if the duration is present). We'll keep this in place until the API stops sending "default" durations of 2 hours. At that time we switch to `DateTimeRange` which only shows an end time if the duration is present.